#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_compatibility_with_rails_2.1.dpatch by  <gwolf@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes compatibility breakage with Rails 2.1

@DPATCH@

diff -urNad libgettext-ruby~/lib/gettext/rails.rb libgettext-ruby/lib/gettext/rails.rb
--- libgettext-ruby~/lib/gettext/rails.rb
+++ libgettext-ruby/lib/gettext/rails.rb
@@ -271,6 +271,9 @@ end
 
 module ActionView #:nodoc:
   class Base #:nodoc:
+    # Needed for Rails >=2.1 compatibilty
+    delegate :file_exists?, :to => :finder unless respond_to?(:file_exists?)
+
     alias render_file_without_locale render_file #:nodoc:
     # This provides to find localized template files such as foo_ja.rhtml, foo_ja_JP.rhtml
     # instead of foo.rhtml. If the file isn't found, foo.rhtml is used.

